### PR TITLE
FEAT CCL support

### PR DESCRIPTION
**Pull request by NeonAndrii**
Addresses #17 

Check whether script exists and emit message to CC to start script

### CODE REVIEW

File: __init__.py [hunk 1]
```diff
@@ -64,6 +64,9 @@ class AlertType(IntEnum):
 
 
 class AlertSkill(MycroftSkill):
+    __location__ = os.path.realpath(
```
Comment (NeonDaniel) [inline __init__.py:67]: Is this required?

File: __init__.py [hunk 2]
```diff
@@ -1213,21 +1239,26 @@ def _alert_expired(self, message):
         if not self.preference_skill(message)["quiet_hours"]:
             self._make_alert_active(alert_time)
         else:
-            self._make_alert_missed(alert_time)
+            # TODO: should we find another way to manage quiet hours for scripts? AP
```
Comment (NeonDaniel) [inline __init__.py:1242]: We should discuss this in our next meeting. Quiet hours in general might deserve further discussion for how we manage turning them off (maybe after a set time, for a certain type of alert, etc.)

File: __init__.py [hunk 3]
```diff
@@ -1236,12 +1267,25 @@ def _notify_expired(self, message):
...
+        LOG.debug(message.data)
+        try:
+            message.data["file_to_run"] = message.data.get("script")
+            # emit a message telling CustomConversations to run a script
+            self.bus.emit(Message("neon.run_alert_script", data=message.data, context=message.context))
+            LOG.info("The script has been executed with CC")
+        except Exception as e:
+            LOG.error(e)
+            LOG.info("The alarm script has expired with an error, notify without added to missed")
+            self._script_error_notify_expired(message)
```
Comment (NeonDaniel) [inline __init__.py:1280]: Just use `_speak_notify_expired`?

File: __init__.py (hunk 1 above)
Comment (NeonAndrii) [reply __init__.py:67]: not at all, I considered using it to duplicate scripts into the skill package, but that does not really make sense

File: __init__.py (hunk 2 above)
Comment (NeonDaniel) [reply __init__.py:1240]: Priority TODO Added #19

**Review by NeonDaniel — APPROVED** [review]

**Pull request merged**